.github/workflows/publish-docker.yml

```diff
@@ -23,13 +23,19 @@ jobs:
         with:
           images: ${{ env.DOCKER_TAG_IMAGE }}
 
-      - name: Build Docker Image
-        run: |
-          docker build -t ${{ steps.meta.outputs.tags }} -f Dockerfile .
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
-      - name: Push to Container Registry
-        if: github.event_name != 'pull_request'
-        run: |
-          echo ${{ secrets.GITHUB_TOKEN }} | docker login -u ${{ github.actor }} https://ghcr.io --password-stdin
-          docker push ${{ steps.meta.outputs.tags }}
+      - name: Login to DockerHub
```

Comment:
ghcr.io is not DockerHub 🙈

Reply:
lol copy 🍝 error. I _did_ make sure to not leave the actual registry as Docker Hub 😅